frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -1,2 +1,2 @@
-export const resourceStatusShouldContain = (desiredStatus: string) =>
-  cy.contains('[data-test="status-text"]', desiredStatus);
+export const resourceStatusShouldContain = (desiredStatus: string, timeout = 30000) =>
+  cy.contains('[data-test="status-text"]', desiredStatus, { timeout });
```

Comment:
`timeout` should be undefined by default so it uses the global command timeout. It would probably be better to take an optional `options` parameter so you can set any of the options.

```suggestion
export const resourceStatusShouldContain = (desiredStatus: string, options) =>
  cy.contains('[data-test="status-text"]', desiredStatus, options);
```

Reply:
Hi, also you removed the default `timeout = 30000` which I believe another snapshot test was relying on.